client/src/command/health.rs

```diff
@@ -0,0 +1,45 @@
+use crate::config::{DEFAULT_NODE_ADDR, DEFAULT_PROTOCOL};
+use crate::node::{HealthResponse, DEFAULT_HEALTH_ENDPOINT};
+use clap::ArgMatches;
+use lazy_static::lazy_static;
+
+lazy_static! {
```

Comment:
Can we set a default endpoint from which the node always listens (e.g. `127.0.0.1:5555`) and pick up the endpoint the client points to for the rpc calls either through an env variable (like `IPC_ENDPOINT_ADDR`) or a config file?

Reply:
there is a config for this and variable defined